addons/binding/org.openhab.binding.rme/src/main/java/org/openhab/binding/rme/RMEBindingConstants.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rme;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RMEBinding} class defines common constants, which are used across
+ * the whole binding.
+ * 
+ * @author Karel Goderis - Initial contribution
+ */
+public class RMEBindingConstants {
+
+	public static final String BINDING_ID = "rme";
```

Comment:
Could you please apply the code formatter to all your code? I see that here are still tabs.